test/parallel/test-net-server-max-connections-close-makes-more-available.js

```diff
@@ -77,9 +83,7 @@ process.on('exit', function() {
   assert.deepEqual(sent, [0, 1, 2, 3]);
   // ...but that only connections 0 and 2 were successful.
   assert.deepEqual(received, [0, 2]);
+  // ... and 1, 3 failed
+  assert.deepEqual(failed, [1, 3]);
 });
 
-process.on('unhandledRejection', function() {
-  console.error('promise rejected');
-  assert.fail(null, null, 'A promise in the chain rejected');
-});
```

Comment:
It might be useful to leave this there in case something late in the promise chain rejects and nothing is there to handle it. (On the other hand, that may be unlikely and outside the scope of this test.)